pkg/controller/accountclaim/reuse.go

```diff
@@ -391,3 +464,19 @@ func matchAccountForReuse(account *awsv1alpha1.Account, accountClaim *awsv1alpha
 	}
 	return false
 }
+
+func deleteAllAccessKeys(reqLogger logr.Logger, client awsclient.Client, userName string) error {
```

Comment:
Just a suggestion, what do you think of naming this function something along the lines of `deleteAllUserAccessKeys`?